src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -390,31 +380,31 @@ object HttpGoogleServicesDAO extends GoogleServicesDAO with FireCloudRequestBuil
                         // generate direct link per https://cloud.google.com/storage/docs/authentication#cookieauth
                         logger.info(s"$userStr download via direct link allowed for [$objectStr]")
                         val redirectUrl = getDirectDownloadUrl(bucketName, objectKey)
-                        RequestCompleteWithHeaders(StatusCodes.TemporaryRedirect, HttpHeaders.Location(Uri(redirectUrl)))
+                        RequestCompleteWithHeaders(StatusCodes.TemporaryRedirect, Location(Uri(redirectUrl)))
 
                     }
                   }
                 }
 
               case _ =>
                 // the user does not have access to the object.
-                val responseStr = objectResponse.entity.asString.replaceAll("\n","")
```

Comment:
this will no longer be logged. consuming the entity to put it in the logs was a PITA but maybe i wasn't going about it the right way